src/Generator.js

```diff
@@ -72,6 +72,9 @@ const getNewPhase = (line) => {
   if (line === '\'\'\'' || line === '"""') {
     return 'DOC_STRING_STARTED';
   }
+  if (line.startsWith('#')){
```

Comment:
So, the idea behind these phases was that they indicate what _(multi-line) section_ the parser is currently looking at in the feature file. The phase can be important context for understanding future lines.  However, at this point for comments they can be parsed line-by-line and we do not really need to communicate any context for future lines. 

Probably can just do this check down in the parseFeatureFile function like this one 
```.js
} else if (line.startsWith('@')) {
```

Reply:
Changed.